Dockerfile

```diff
@@ -22,8 +22,8 @@ WORKDIR /opt/letsencrypt
 # directories in its path.
 
 
-COPY bootstrap/ubuntu.sh /opt/letsencrypt/src/ubuntu.sh
-RUN /opt/letsencrypt/src/ubuntu.sh && \
+COPY letsencrypt-auto-source/letsencrypt-auto /opt/letsencrypt/src/letsencrypt-auto
```

Comment:
Since we're not instructing users to make the shift to the new `letsencrypt-auto` yet, I don't think we should be making the changes to the Dockerfile now.